### ISSUE TITLE
Receiving a 400 responce on stopLatestPlan()

### ISSUE THREAD

**Issue by scholzi100** [lagmoellertim/freenet-funk-api#8]
It looks like I found a problem in the library while playing around with it. I noticed `stopLatestPlan()` always return a 400 staus code. So I digged a bit deeper.

I checked if there was a tariff on my account and there was:
```
{
    'id': '6236bfef-a85c-444e-b978-84e2bd574766',
    'booked': '2019-08-17T11:47:33.848Z',
    'starts': '2019-08-17T22:00:00.000Z',
    'state': 'READY',
    'productServiceId': '8',
    'productServiceInfo': {
        'id': '8',
        'label': 'unlimited',
        'follower': None,
        'marketingInfo': {
            'name': 'unlimited',
            '__typename': 'MarketingInfo'
        },
        '__typename': 'ProductService'
    },
    '__typename': 'TariffCustomerProductService'
}
```

By calling stopLastPlan() the content received always is: 
```
'{
    "errors":[
        {
            "message":"Cannot query field \\"TariffFragment\\" on type \\"TariffCustomerProductService\\".",
            "locations":[
                {
                    "line":4,
                    "column":5
                }
            ],
            "extensions":{}
        },
        {
            "message":"Fragment \\"TariffFragment\\" is never used.",
            "locations":[
                {
                    "line":9,
                    "column":1
                }
            ],
            "extensions":{}
        }
    ]
}'
```

As stated in `graphql_schema.py` this must be the TariffFragment which is not set:
```
""",
    "remove_product": """
mutation TerminateTariffMutation($tariffID: String!) {
  tariffTerminate(customerProductServiceId: $tariffID) {
    TariffFragment
    __typename
  }
}

fragment TariffFragment on TariffCustomerProductService {
  id
```

I tried to replace `TariffFragment` in `TerminateTariffMutation` with `...TariffFragment`. But this poduced a reply with status code 200 and following content: 
```
{
    "errors":[
        {
            "message":"ABE-E0119: ErrCPSTerminateNotAllowedViaHandler",
            "locations":[
                {
                    "line":3,
                    "column":3
                }
            ],
            "path":[
                "tariffTerminate"
            ],
            "extensions":{}
        }
    ],
    "data":null
}'
```

To me it look like the server tries to prevent this action or something.

(refactord replies for readability)

**Comment by tc-maxx** [comment]
The three points were missing :)

**...**TariffFragment

```
mutation TerminateTariffMutation($tariffID: String!) {
  tariffTerminate(customerProductServiceId: $tariffID) {
    ...TariffFragment
    __typename
  }
}
```

**Comment by scholzi100** [comment]
Fixed yet?

**Comment by scholzi100** [comment]
Closed by #14
